here-naksha-lib-psql/src/commonMain/kotlin/naksha/psql/executors/write/WriteFeatureUtils.kt

```diff
@@ -80,7 +80,7 @@ internal object WriteFeatureUtils {
     internal fun tuple(
         storage: PgStorage,
         tupleNumber: TupleNumber,
-        feature: NakshaFeature,
+        feature: NakshaFeature?,
```

Comment:
This is not necessary, stick with feature being **not null**! See above in review why!